TWLight/users/views.py

```diff
@@ -690,9 +688,15 @@ def get_context_data(self, **kwargs):
                     each_authorization.about_to_expire
                     or not each_authorization.is_valid
                 ):
-                    each_authorization.latest_app = each_authorization.get_latest_app()
-                    if each_authorization.latest_app:
-                        if not each_authorization.latest_app.is_renewable:
+                    latest_app = each_authorization.get_latest_app()
+                    if latest_app:
+                        if latest_app.status != Application.SENT:
+                            each_authorization.latest_sent_app = (
+                                each_authorization.get_latest_sent_app()
+                            )
+                        else:
+                            each_authorization.latest_sent_app = latest_app
+                        if not latest_app.is_renewable:
```

Comment:
Do we need to do all this? Why can't we just use `get_latest_sent_app()` right away?

Reply:
I did think about this and [apparently](https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/applications/models.py#L390) APPROVED applications can be renewed too. So, didn't want to break anything, and open to suggestions.

Reply:
Also, we cannot stop if we don't find any SENT apps. We'd want to know to know if there are other apps that fit our criteria for which we'd need `get_latest_app()` here.

Reply:
Well that's weird. I don't know why we would want to let folks renew an approved application. Will ping Nikki on Slack, we might want to adjust that.